daffodil-test/src/test/scala/org/apache/daffodil/section07/variables/TestVariables.scala

```diff
@@ -93,6 +93,8 @@ class TestVariables {
 
   @Test def test_unsignedIntVarCast(): Unit = { runner.runOneTest("unsignedIntVarCast") }
 
+  @Test def test_choiceBranchVariables(): Unit = { runner.runOneTest("choiceBranchVariables") }
```

Comment:
If this test fails you want to commit with the test commented out and // DAFFODIL-2375 comment.

Reply:
Right, will do